designcompose/src/androidTest/kotlin/com/android/designcompose/JniNoNetworkTests.kt

```diff
@@ -48,7 +48,7 @@ class JniNoNetworkTests {
         val startTime = SystemClock.elapsedRealtime()
         while (
             connectivityManager.activeNetwork != null &&
-                SystemClock.elapsedRealtime() - startTime < 2000
+                SystemClock.elapsedRealtime() - startTime < 20000
```

Comment:
Did you find it takes this long sometimes?

Reply:
Yeah. Honestly, I'm thinking of giving up on this check, I'm getting occasional failures in other CI runs too. Basically, there isn't a "proper" way to run a instrumented test with no network access, and these alternative ways I'm trying aren't consistently working. It's not that important of a test anyways.